cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=False)
+    description = serializers.CharField(required=False)

```

Comment:
The `description` field is also marked as `required=False`. Consider whether this field should be optional or required based on your application's needs.